daffodil-test/src/test/scala/org/apache/daffodil/section05/simple_types/TestSimpleTypesUnparse.scala

```diff
@@ -50,7 +50,9 @@ class TestSimpleTypesUnparse {
   @Test def test_hexBinary_unparse_10(): Unit = { runner.runOneTest("hexBinary_unparse_10") }
   @Test def test_hexBinary_unparse_11(): Unit = { runner.runOneTest("hexBinary_unparse_11") }
   @Test def test_hexBinary_unparse_12(): Unit = { runner.runOneTest("hexBinary_unparse_12") }
+  @Test def test_hexBinary_unparse_13(): Unit = { runner.runOneTest("hexBinary_unparse_13") }
```

Comment:
At the top of this we have this test commented out. That can be removed now.

Reply:
Thanks. I missed seeing that.